odoo-shippable/scripts/build-image.sh

```diff
@@ -241,8 +242,13 @@ endif
 EOF
 
 cat >> ~/.vimrc.before.local << EOF
+# Disable youcompleteme for default
+#let g:spf13_bundle_groups = ['general', 'writing', 'odoovim', 'wakatime',
+#                           \ 'programming', 'youcompleteme', 'php', 'ruby',
+#                           \ 'python', 'javascript', 'html',
+#                           \ 'misc']
 let g:spf13_bundle_groups = ['general', 'writing', 'odoovim', 'wakatime',
-                           \ 'programming', 'youcompleteme', 'php', 'ruby',
```

Comment:
if we just disable this line could be enought to avoid the current error. Rigth?
If someone want to enable it is so easy just add this line